TestsExplicit/TestsExplicit.csproj

```diff
@@ -3,10 +3,20 @@
     <TargetFrameworks>net472;netcoreapp2.2</TargetFrameworks>
     <DisableFody>true</DisableFody>
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="xunit.runner.json" />
+    <Content Include="xunit.runner.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="XunitContext" Version="1.8.0" />
     <PackageReference Include="Verify.Xunit" Version="1.14.1" />
     <PackageReference Include="FodyHelpers" Version="6.1.0" />
+    <PackageReference Include="Verify.Xunit" Version="1.13.0" />
+    <PackageReference Include="XunitContext" Version="1.6.0" />
+    <PackageReference Include="Verify.Xunit" Version="1.12.2" />
+    <PackageReference Include="FodyHelpers" Version="6.1.0" />
```

Comment:
- `Verify.Xunit` is present 3 times
- `XunitContext` is included twice
- `FodyHelpers` is included twice